### ISSUE TITLE
Fix request: false positive for SimpleKey in AvoidIdentityCacheKeys

### ISSUE THREAD

**Issue by stokpop** [jborgers/PMD-jPinpoint-rules#575]
```java
class Foo implements KeyGenerator {
    @NotNull @Override
    public Object generate(Object target, Method method, Object... params) {
        return new SimpleKey(target.getClass().getName(), method.getName(), params); // <-- false positive
    }
}
```

This is the code from test case "no violation: issue #149 - false positive for AvoidIdentityCacheKeys when used SimpleKey", but that does not create the hit because of missing imports:

```java
import org.springframework.cache.interceptor.KeyGenerator;
import org.springframework.cache.interceptor.SimpleKey;
```

Adding the imports makes the unit test fail.
